main.mf

```diff
@@ -36,7 +44,22 @@ SRCS =  esp82xx/fwsrc/uart.c \
 	user/user_main.c
 
 LIBS = main lwip ssl upgrade net80211 wpa phy lwip pp crypto
-INCL = $(SDK)/include esp82xx/include .
+
+ifeq ($(strip $(DOCKER)), yes)
+	INCL = $(SDK)/include $(DOCKER_PREFIX)/esp82xx/include $(DOCKER_PREFIX)
+	SRCS =  $(DOCKER_PREFIX)/esp82xx/fwsrc/uart.c \
```

Comment:
@con-f-use ... how would you collapse those with a Makefile `$(foreach)` loop? It's just adding a prefix to each `.c` in `SRCS` :/

This c&p feels a bit too ugly to leave it as it is now.